sql/activity_stream_bi/impression_stats_by_experiment_v1/init.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE TABLE
+  activity_stream_bi.impression_stats_by_experiment_v1
+PARTITION BY
+  DATE(submission_timestamp)
+CLUSTER BY
+  experiment_id
+OPTIONS
+  (
+    require_partition_filter = TRUE, --
+    partition_expiration_days = 180
+  )
+AS
+SELECT
+  submission_timestamp,
+  experiment.key AS experiment_id,
+  experiment.value.branch AS experiment_branch,
```

Comment:
I know that your existing queries coalesce experiment info with `shield_id`. I assume the `shield_id` variants are no longer relevant. Is that correct?

Is there a particular amount of backfill that's relevant here? Would backfilling a month or two be sufficient, or do you have use cases where you need to look back on the full 6 months?

Reply:
Yup, `shield_id` is in there from the period where we had a mix of data with `shield_id`/`experiments`.

One month of backfill should be sufficient!